Rewrite AGCDN guide and announce new SKU (#9628)
pantheon-systems/documentation: source/content/guides/agcdn/01-introduction.md

Each AGCDN service configuration is offered as an annual contract subscription. Customers can add multiple sites to their AGCDN configuration, or conditionally apply specific rules.

<Alert title="Note" type="info" >

> At the moment, AGCDN only works with custom domains. `.pantheonsite.io` domains are not covered.

Author: The copy here in this alert was existing on this page, I think it's fair to remove "At the moment" though 

```suggestion
AGCDN only works with custom domains. Platform domains (e.g., `live-site-name.pantheonsite.io`) are not covered.
```

> At the moment, AGCDN only works with custom domains. `.pantheonsite.io` domains are not covered.

Reviewer: @jazzsequence agreed here. As of right now, I have no plans to change this. (this may change in the future but will be updated accordingly)

@rachelwhitton

</Alert>

## AGCDN Packages 

### IO Starter
_Rich Content from Globally Distributed Points of Presence_

Improve Lighthouse scores and core web vitals with improved page load times. Pantheon’s IO Starter package brings modern image optimization and auto WebP support to render pages quickly at the edge, taking load off your application and adding resilience to your site.

### Security Starter
_Improve Security & Maintain Compliance_

As attack surfaces expand, businesses require additional security beyond the CMS to protect their web sites. Pantheon’s Security Starter brings industry standard rules including OWASP and Drupal/WordPress protections. Secure your sites with a web application firewall (WAF) - tuned for each CMS, geolocation and IP blocking.

### Security Starter with Rate Limiting

> Security Starter is also available with a rate-limiting upgrade. Rate-limiting provides a layer of defense against DDoS attacks and high-traffic events for websites by throttling requests to be under specific thresholds, at the Fastly Edge Layer.

Author: @conorbauer23 can you review and advise?

> Security Starter is also available with a rate-limiting upgrade. Rate-limiting provides a layer of defense against DDoS attacks and high-traffic events for websites by throttling requests to be under specific thresholds, at the Fastly Edge Layer.

Reviewer 2: Just a link to Fastly in general. We refer to it, but if you didn't know it was part of the stack, it's just a technical term without a definition. A link to the Fastly home page on the first time we mention it for context would be enough, IMO.

> Security Starter is also available with a rate-limiting upgrade. Rate-limiting provides a layer of defense against DDoS attacks and high-traffic events for websites by throttling requests to be under specific thresholds, at the Fastly Edge Layer.

Author: done!
